docs/src/pages/demos/tables/ReactVirtualizedTable.tsx

```diff
@@ -0,0 +1,228 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { withStyles, Theme, createStyles } from '@material-ui/core/styles';
+import TableCell from '@material-ui/core/TableCell';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import Paper from '@material-ui/core/Paper';
+import { AutoSizer, Column, SortDirection, Table } from 'react-virtualized';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    flexContainer: {
+      display: 'flex',
+      alignItems: 'center',
+      boxSizing: 'border-box',
+    },
+    tableRow: {
+      cursor: 'pointer',
+    },
+    tableRowHover: {
+      '&:hover': {
+        backgroundColor: theme.palette.grey[200],
+      },
+    },
+    tableCell: {
+      flex: 1,
+    },
+    noClick: {
+      cursor: 'initial',
+    },
+  });
+
+class MuiVirtualizedTable extends React.PureComponent {
+  static propTypes = {
```

Comment:
I am unsure if this is the correct method for handling `propTypes` because I was getting the error:

'propTypes' does not exist on type 'typeof MuiVirtualizedTable'